server/app/com/framed/playscalajs/controllers/Application.scala

```diff
@@ -9,7 +9,8 @@ import play.api.mvc._
 class Application @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
 
   def index = Action {
-    Ok(views.html.index(SharedMessages.itWorks))
+    val image = "/versionedAssets/images/frontpage.jpg"
```

Comment:
The variable name `image` is pretty liberal in this context. Would recommend to use a name that explains the purpose of the variable more e.g. `imageURL` or `indexImageLocation`. 

Although your implementation works, it may be difficult in the future to spot that the index has an image embedded. A different approach could be to pull the `val` into the Application and make `image` an argument to the `index` method. That way it may be clearer in future iterations that index depends on a static image.

Reply:
Ok did it: 
a19b407 although it got too complicated to make image an argument of index since index does not take arguments in routes.